pkg/app/piped/driftdetector/ecs/detector.go

```diff
@@ -299,6 +305,11 @@ func ignoreParameters(liveManifests provider.ECSManifests, headManifests provide
 		liveService.NetworkConfiguration = &types.NetworkConfiguration{AwsvpcConfiguration: &awsvpcCfg}
 	}
 
+	// Ignore the diff of DeploymentConfiguration if it's not specified.
```

Comment:
Would you add `- service.DeploymentConfiguration` to the function's comment?

```go
// TODO: Maybe we should check diff of following fields when not set in the head manifests in some way. Currently they are ignored:
//   - service.DeploymentConfiguration                   <--------- HERE
//   - service.PlatformVersion
//   - service.RoleArn
func ignoreParameters(liveManifests provider.ECSManifests, headManifests provider.ECSManifests) (live, head provider.ECSManifests) {
```

Reply:
added in https://github.com/pipe-cd/pipecd/pull/5344/commits/930b46fc7f1fdf5523793cb7d503330cbdd6e7bb